CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 * Fixed an issue where `UserPuckCourseView`’s color desaturated during turn-by-turn navigation even as the location was being updated. ([#3836](https://github.com/mapbox/mapbox-navigation-ios/pull/3836))
 * `UserPuckCourseView` no longer desaturates its color based on the age of the last location update. `RouteController` simulates location updates whenever Location Services is unable to receive real location updates. To ensure a steady stream of location updates outside of turn-by-turn navigation, install a `PassiveLocationManager`. ([#3836](https://github.com/mapbox/mapbox-navigation-ios/pull/3836))
+* Fixed the image cache loss when App goes to background. ([#3840](https://github.com/mapbox/mapbox-navigation-ios/pull/3840))
```

Comment:
```suggestion
* Fixed an issue where shields disappeared after the application returns to the foreground. ([#3840](https://github.com/mapbox/mapbox-navigation-ios/pull/3840))
```

Reply:
Thanks for the suggestion, I'll update the ChangeLog.